src/screens/OtherProfile/index.js

```diff
@@ -85,7 +82,7 @@ const OtherProfile = () => {
   const [loadingGenerateAnon, setLoadingGenerateAnon] = React.useState(false);
   const [anonProfile, setAnonProfile] = React.useState();
   const {mappingColorFeed} = useCoreFeed();
-  const {updateFollowDiscoveryContext, getIsMeFollowingTargetStatus} = useDiscovery();
+  const {updateFollowDiscoveryContext} = useDiscovery();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getIsMeFollowingTargetStatus` has been removed from the destructuring of `useDiscovery()`. If this function is no longer used in this component, this change is fine. However, if it's still needed elsewhere in the component, you should add it back.



<!-- This is an auto-generated comment by OSS CodeRabbit -->